pages/creating-a-campaign/rbi-campaign.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Creating a campaign with a Range Based Incentive
+description: How to create a campaign with Range Based Incentive
+---
+
+## Creating a campaign with Range Based Incentive
+
+Creating a basic campaign on Metrom is straightforward, as covered in
+[creating a campaign](/creating-a-campaign). Launching one with an attached Range-based Incentive is just as easy. For a quick refresher on KPIs, check out [Range Based Incentives](/rbi).
+
+
+
+### Configuration
+
+Once all the required form steps are completed, it's time to enable the **SET RANGE TO REWARD**  and
+explore the available configuration options.
+
+Once enabled, The interface shows a range visualizer with the current price displayed at the top. 
+
+<div className="mediaWrapper">
+    ![Empty RBI step](/images/rbi-config.png)
+</div>
+
+
+- **Current Price**: Shows you the current price for the tokens that are present in the selected pool. You can toggle relative price wrt to Token A / Token B by clicking on the token name.  
```

Comment:
I'd write this without the `wrt` shortnahd:

`You can toggle the relative price in terms of Token A / Token B by clicking on the token name.`